app/assets/stylesheets/new_book_suggestion.scss

```diff
@@ -0,0 +1,42 @@
+.new-form {
+  margin-left: 50px;
+  margin-top: 50px;
+  width: 280px;
+}
+
+.new-form-item {
+  justify-content: space-between;
+  margin-top: 20px;
+  width: 100%;
+
+  input {
+    border-bottom: 1px solid $gray;
+  }
+
+  select {
+    border-radius: 5px;
+  }
+}
+
+.new-form-bottom {
+  margin-top: 25px;
+  align-self: flex-end;
+
+  input {
+    background-color: $orange;
+    border-radius: 5px;
+    height: 40px;
+    width: 200px;
+  }
+}
+
+#suggestion-table {
```

Comment:
Don't use ids in CSS (even though it's ok for grabbing an element in javascript). They have higher priority than simpler rules and that can cause trouble when adding new style to a component.